src/amoc.app.src

```diff
@@ -7,6 +7,7 @@
                   kernel,
                   stdlib,
                   lager,
+                  jiffy,
```

Comment:
I know we need `jiffy` because of `trails`. We also have `jsx` which does pretty much the same as `jsx`. Could you check if we can replace `jsx` with `jiffy`? If so, then we should do this in a separate PR.